tests/e2e/lib/page-helper.js

```diff
@@ -54,8 +54,8 @@ export async function waitForSelector( page, selector, options = {} ) {
  * @param {Object} options Custom options to modify function behavior.
  */
 export async function waitAndClick( page, selector, options = { visible: true } ) {
-	await waitForSelector( page, selector, options );
-	return await page.click( selector, options );
+	const element = await waitForSelector( page, selector, options );
```

Comment:
I changed this to use the element returned by `waitForSelector()`, as I'd originally modified `waitForSelector()` to accept an XPath selector, too (9f62cb1). That's since been reverted, I have no strong feelings either way about this change.